src/state/ling.js

```diff
@@ -26,12 +26,24 @@ const ling = merge(cloneDeep(jobBase), cloneDeep(jobSingleAction), {
 					Object.keys(action.requiredItems).forEach(key => {
 						if (key.toLowerCase().includes('meat')) {
 							action.requiredItems[key] -= 1;
-							if (action.requiredItems[key] == 0) delete action.requiredItems[key];
+							if (action.requiredItems[key] == 0) delete action.requiredItems[key]; // todo ask about this for xenobio tool

```

Comment:
i kind of want to completely replace the resource cost for the xenobio tool, this is the only thing existing in the game that modifies the input of a task afaik so i should look at it later

Reply:
Could just throw it in a if potionItemId !="toolLing"?

Reply:
Already fixed, I just forgor to remove this comment 💀